### ISSUE TITLE
Fix for Repentance Online Beta

### ISSUE THREAD

**Issue by biphelps2** [Zamiell/isaac-save-viewer#2]
Hi, thanks for all the work done on this project!

The Repentance online beta (https://store.steampowered.com/news/app/250900/view/3892861608043687818) adds a few extra achievements to the game (I think for showing the "you just won an online game!" message), which stop this tool from working. It will report "Failed to find the achievement for ID x" and will not generate the tables.

I have a commit in a fork (https://github.com/biphelps2/isaac-save-viewer/commit/da0b6a61c2e7e522aef15365ed975841723a9d45) which adds a new "unknown achievement" to `achievements.json` (to match the existing "unknown item" in `items.json`), and generates the tables with "unknown" rows as required.

(I thought that improving robustness was a better approach than adding some possibly-temporary items to the data)

If you're interested in a pull request, I'm happy to create one - just wanted to quickly check it was ok to send through.

Thanks again!

**Comment by Zamiell** [comment]
yes please, send a PR, also you can msg me on Discord if you have any problems

**Comment by biphelps2** [comment]
Great - PR created here: #3

**Comment by Zamiell** [comment]
do you want to do a PR for the achievements now?

**Comment by biphelps2** [comment]
As in collectables? Can do - the online beta doesn't add any new ones so save files work fine now, but happy to make things consistent

**Comment by Zamiell** [comment]
um, i didn't mean collectibles, but yeah a PR for the collectibles would be fine, in order to make the code more consistent.

i meant to update the "achievements.json" file with the new achievements.

**Comment by biphelps2** [comment]
I don't know what all the new achievements are unfortunately, haven't unlocked them all + since they might be temporary and aren't listed on the wiki, I think it makes sense to leave them out for now.

**Comment by Zamiell** [comment]
ok sounds good
